### PR TITLE
Bug/dev 3291 fix ada conflicts

### PR DESCRIPTION
**Pull request by pascalachkar**
<!--

# 🎉 Thanks for taking the time to contribute to Findify! 🎉

It is highly appreciated that you take the time to help improve Findify.
We appreciate it if you would take the time to document your Pull Request.

Before submitting a new PR please read the [CONTRIBUTING.md](./CONTRIBUTING.md).

### CODE REVIEW

File: packages/react-components/src/components/Cards/Content/index.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default ({
   item,
   config,
   theme = styles,
-  Container = 'div',
+  Container = 'a',
```
Comment (timaramazanov) [inline packages/react-components/src/components/Cards/Content/index.tsx:31]: @pascalachkar let's remove `Container` prop at all. Also pls provide `href` attribute to the `a` element (it should be equal to `item.get('url')`)

File: packages/react-components/src/components/Cards/Product/index.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default ({
   theme = styles,
   className,
   config,
-  Container = 'div',
+  Container = 'a',
```
Comment (timaramazanov) [inline packages/react-components/src/components/Cards/Product/index.tsx:53]: @pascalachkar let's remove Container prop at all. Also pls provide href attribute to the a element (it should be equal to item.get('product_url'))

File: packages/react-components/src/components/Cards/Content/index.tsx (hunk 1 above)
Comment (pascalachkar) [reply packages/react-components/src/components/Cards/Content/index.tsx]: Done

File: packages/react-components/src/components/Cards/Product/index.tsx (hunk 2 above)
Comment (pascalachkar) [reply packages/react-components/src/components/Cards/Product/index.tsx]: Done

**Review by timaramazanov — APPROVED** [review]
lgtm

**Pull request merged**